package.xml

```diff
@@ -9,16 +9,15 @@
 
   <buildtool_depend>cmake</buildtool_depend>
 
-  <depend>KimeraRPGO</depend>
-  <depend>GTSAM</depend>
-  <depend>gflags</depend>
-  <depend>glog</depend>
-  <depend>opencv</depend>
-  <depend>opengv</depend>
   <depend>dbow2</depend>
+  <depend>gtsam</depend>
+  <depend>kimera_rpgo</depend>
+  <depend>libgflags-dev</depend>
+  <depend>libgoogle-glog-dev</depend>
```

Comment:
I think the two above are irrelevant actually because we use cmake files to find glog/gflags as I had tons of issues with these two in multiple systems...

Reply:
> use cmake files to find glog/gflags

Not sure being explicit about this external dependency the manifest would change what the CMake does to find the glog/gflags installed in the cmake path. Are you saying that the system install of glog/gflags in the latest ubuntu LTS is still an issue?